app/controllers/AdminController.php

```diff
@@ -59,6 +59,7 @@ public function  showReorderCategory() {
 
 	public function addCategory() {
 		$name = Input::get('name');
+		$kakas = "kakas";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "$kakas" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=php%3AS1481)